velox/common/base/SkewedPartitionBalancer.h

```diff
@@ -85,13 +97,15 @@ class SkewedPartitionRebalancer {
   };
 
   Stats stats() const {
-    return stats_;
+    return Stats{
+        .numBalanceTriggers = numBalanceTriggers_.load(),
+        .numScaledPartitions = numScaledPartitions_.load()};
   }
 
  private:
-  bool shouldRebalance() const;
+  bool shouldRebalance(int64_t processedBytes) const;
```

Comment:
Maybe uint64_t?

Reply:
I used int64_t as some operation might lead to negative value under concurrent update with single lock. So use int64_t to keep it simple.